ReleaseProcessAutomation/Scripting/MSBuildCallAndCommit.cs

```diff
@@ -57,7 +58,14 @@ public int CallMSBuildStepsAndCommit (MSBuildMode msBuildMode, SemanticVersion v
     if (string.IsNullOrEmpty(msBuildPath))
     {
       _log.Warning("No MSBuild Path specified in config, will continue without MSBuild");
-      _console.WriteLine("There was no MSBuildPath specified in the config, will continue without Invoking MSBuild");
+      _console.WriteLine("There was no MSBuildPath specified in the config\nWill continue without invoking MSBuild");
+      return -1;
+    }
+
+    if (!File.Exists(msBuildPath))
+    {
+      _log.Warning("The configured MSBuildpath does not exist");
+      _console.WriteLine("The configured MSBuildPath does not exist\nPlease configure a proper MSBuildPath in the config\nWill continue without invoking MSBuild");
```

Comment:
Please print the msbuild path in both the console and the log message, otherwise you have to dig and reverse-engineer if you think your path is correct, but isn't. Use single-quotes for delimit the path value in the message or place it in parenthesis. (i.e. the style of the delimiter depends on how you format the message)